Formula/codeclimate.rb

```diff
@@ -1,19 +1,18 @@
-require "formula"
-
 class Codeclimate < Formula
-  CODECLIMATE_VERSION = "0.24.2".freeze
-
+  desc "Code Climate CLI"
   homepage "https://github.com/codeclimate/codeclimate"
-  version CODECLIMATE_VERSION
-
-  url "https://github.com/codeclimate/codeclimate/archive/v#{CODECLIMATE_VERSION}.tar.gz"
-  sha1 "c28ce24aea878ecd8b1bd65cc313206bcd36aece"
+  url "https://github.com/codeclimate/codeclimate/archive/v0.24.2.tar.gz"
+  sha256 "f6ab47d602c8a6ec2d4e9f86df829cc31c3059ad025058c7646499b154242351"
 
   def install
     # Alter PATH to ensure `docker' is available
-    system "env", \
-      "PATH=#{HOMEBREW_PREFIX}/bin:#{ENV["PATH"]}", \
-      "PREFIX=#{prefix}", "make", "install"
+    if Formula["docker"].linked_keg.exist?
```

Comment:
> can you get them to provide the output of `brew doctor`?

This bit seems relevant:

```
Warning: You have unlinked kegs in your Cellar
Leaving kegs unlinked can lead to build-trouble and cause brews that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
    docker
```

However, when she runs `brew link docker` there's an error because `/usr/local/bin/docker` already exists.

```
Linking /usr/local/Cellar/docker/1.9.1... 
Error: Could not symlink bin/docker
Target /usr/local/bin/docker
already exists. You may want to remove it:
  rm '/usr/local/bin/docker'

To force the link and overwrite all conflicting files:
  brew link --overwrite docker

To list all files that would be deleted:
  brew link --overwrite --dry-run docker
```

> they should already have `/usr/local/bin` in their path

It's likely they do (in a real shell). In our experience, the Formula logic runs in another context with a different `$PATH`, one that doesn't include that directory. That's why we added [this](https://github.com/codeclimate/homebrew-formulae/pull/23/files/c8d50494e76121a286e2c415d513ccf192004ad6#diff-7c85e74186616b7da55ec91b684b76cdL15) originally.